### PR TITLE
feat(BE-60): Parse CSV for URLs

### PR DESCRIPTION
**Pull request by noornee**
**Linear Ticket:** 
- https://linear.app/team-descripto/issue/BE-60/parse-csv-for-urls

**Details:**
-   Created a function that parses a csv file and extracts all the valid image urls from the csv
-    made a http Handler that returns all the parsed urls when the csv file is attached

**Test Endpoint:**
to test my endpoint, you need to be logged in to get an access token.
send a post request to this https://discripto.hng.tech/api1/api/v1/login  using this same exact user(johndoe)
i.e.

```
curl https://discripto.hng.tech/api1/api/v1/login \
	--json '{"email": "johndoe@email.com", "password": "password"}'
```

this would return an object containg the access token.

send a POST request to localhost:9000/api/v1/mine-service/upload-csv
```
 curl --silent localhost:9000/api/v1/mine-service/upload-csv \
                -F file=@<path_to_csv_file> \
                -H "Accept: multipart/form-data" \
                -H "Authorization: Bearer <access_token>" | jq 
```

**Preview:**
2022-12-02-000805_1920x684_scrot

### CODE REVIEW

File: pkg/router/mine-service.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ func MineServiceUpload(r *gin.Engine, validate *validator.Validate, ApiVersion s
 		authUrl.POST("/mine-service/upload", mineservice.MineImageUpload)
 		authUrl.POST("/mine-service/url", mineservice.MineImageUrl)
 		authUrl.GET("/mine-service/get-all", mineservice.GetMinedImages)
+		authUrl.POST("/mine-service/upload-csv", mineservice.ParseUrlFromCsv)
```
Comment (emmrys-jay) [inline pkg/router/mine-service.go]: @noornee @usmansadik Does this task need an endpoint to be exposed?

**Comment by emmrys-jay** [comment]
@noornee @usmansadik 

Are you sure this task needed a handler?

File: pkg/router/mine-service.go (hunk 1 above)
Comment (noornee) [reply pkg/router/mine-service.go]: no it doesn't, I'm gonna make changes to my code and revert the commit i made for this!

Comment (emmrys-jay) [reply pkg/router/mine-service.go]: Okay then

Comment (noornee) [reply pkg/router/mine-service.go]: @Emmrys-Jay, reverted the commit already

**Comment by emmrys-jay** [comment]
Looks good to merge

**Comment by kataszn** [comment]
I'm surprised
no it does not
it's not supposed to be a endpoint self

**Comment by kataszn** [comment]
> @noornee @usmansadik 
> 
> Are you sure this task needed a handler?
> 

no it definitely doesn't

**Pull request merged**